1.0/profile.ttl

```diff
@@ -0,0 +1,63 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX prof: <http://www.w3.org/ns/dx/prof/>
+PREFIX role: <http://www.w3.org/ns/dx/prof/role/>
+PREFIX sdo: <https://schema.org/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+
+<http://www.opengis.net/def/geosparql>
+    a prof:Profile ;
+    dcterms:identifier "geosparql" ;
```

Comment:
A `dcterms:identifier` is supposed to be "An unambiguous reference to the resource within a given context". This is ambiguous, because the profile is not the specification and the version identifier is missing.

The examples in https://w3c.github.io/dx-prof/prof use `rdfs:label` instead of `dcterms:identifier.` So I suggest changing this to `rdfs:label "Profile of GeoSPARQL 1.0"@en`.

Reply:
The profiles do also have `dcterms:title` values abd the specific reason for the `dcterms:identifier` property was for use in cataloguing but perhaps a version IRI is needed since we will have 2 profiles for GeoSPARQL, 1.0 & 1.1, with potentially the same IRI. So I've removed `dcterms:identifier` and added `owl:versionIRI`.